src/main.ts

```diff
@@ -242,4 +242,4 @@ async function run(): Promise<void> {
   }
 }
 
-run();
+run().catch(e => core.setFailed(e.message));
```

Comment:
It would be nice if we caught the "Body too long" error specifically and posted a comment about it since the user can't actually do much about their diff being long

Reply:
https://github.com/quizlet/argocd-diff-action/issues/8